e2e-tests/constants/field-ids/insurance/exporter-business/index.js

```diff
@@ -28,6 +28,14 @@ export const EXPORTER_BUSINESS = {
     PERCENTAGE_TURNOVER: 'exportsTurnoverPercentage',
   },
   BROKER: {
+    BROKER_HEADING: 'broker',
```

Comment:
I don't think should have `_HEADING` here as this is the actual radio input right? `isUsingBroker`?

Maybe the parent object should be `USING_BROKER` so we don't have `BROKER.BROKER`

Thoughts?